AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -650,6 +653,17 @@ - (void)scrollViewWillEndDragging:(UIScrollView *)scrollView withVelocity:(CGPoi
   }
 }
 
+- (void)scrollViewDidScroll:(UIScrollView *)scrollView{
+    for (ASCellNode *node in _cellsForVisibilityUpdates) {
+        if (node.shouldObserveVisibility) {
```

Comment:
We could actually keep this property, but I would not expose it as public API. Instead, we should internally set the value of this property based on whether either the cell subclass or its layout delegate implement the method. In order to be able to collect here, without it being in the public header, you could add a @interface ASCellNode (Internal) at the top of this file… or look for another file where it Would be appropriate to put this kind of internal method